README.md

```diff
@@ -1,20 +1,24 @@
 # Rancher Product Docs
 
-Welcome to the Rancher Product docs. We are currently a work in progress. 
+Welcome to the Rancher Product docs.
 
-🚧🛠️👷🛠️🚧
+This repo is used to migrate content from the [Rancher docs](https://ranchermanager.docs.rancher.com/) site to publish it on the [SUSE Documentation site](https://documentation.suse.com/). It runs on the [Antora](https://antora.org/) framework.
 
-We are using this repo to migrate the current [Rancher docs](https://ranchermanager.docs.rancher.com/) site to run on the [Antora](https://antora.org/) framework. 
+See the [Rancher software](https://github.com/rancher/rancher) repo if you have questions or requests for the Rancher platform.
 
 ## Contribute to Rancher Docs
 
-If you're looking to update the currently published iteration of the Rancher docs site, please visit [https://github.com/rancher/rancher-docs](https://github.com/rancher/rancher-docs).
+If you are looking to update the currently published iteration of the Rancher docs site, please visit [https://github.com/rancher/rancher-docs](https://github.com/rancher/rancher-docs).
 
-See the [Rancher software](https://github.com/rancher/rancher) repo if you have questions or requests for the Rancher platform.
+To start contributing, first install the Antora environment by running `make environment`.

```

Comment:
We should use the underlying commands in our instructions instead of the make commands because of (the lack of) Windows compatibility. I've shared reference material with you via Slack.